projects/plugins/boost/app/assets/src/js/pages/settings/elements/MinifyMeta.svelte

```diff
@@ -0,0 +1,124 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte';
+	import { __, sprintf } from '@wordpress/i18n';
+	import PencilIcon from '../../../svg/pencil.svg';
+
+	export let id: string;
```

Comment:
Rather than requiring the component's parent to specify a unique id for internal use within the component, it would be better for the component to invent its own unique id.

You could do this very easily with a [module-level script](https://svelte.dev/tutorial/sharing-code) to keep a counter.

e.g.: 
```
<script lang="ts" context="module">
  let nextId = 0;
</script>

<script lang="ts">
  ...
  const id = nextId++;
  ...
```

That way, users of this component get a simpler interface to work with. :)

Reply:
🤯 I love this - thank you!

Updated in d6a46b37cf1e5921e37c4e7911c366949be90666.